### PR TITLE
Update quick-start for 1-line deployment.

### PR DESCRIPTION
**Pull request by Ben-PH**
See also https://github.com/seed-rs/seed-quickstart/pull/19

### CODE REVIEW

File: crate/guides/quickstart.md [hunk 1]
```diff
@@ -30,17 +30,18 @@ This framework requires you to install [Rust](https://www.rust-lang.org/tools/in
 
 You'll need a recent version of Rust: `rustup update`
 
-The wasm32-unknown-unknown target: `rustup target add wasm32-unknown-unknown`
-
-And cargo-make: `cargo install --force cargo-make`
+Once that's done, you'll need cargo-make to build: `cargo install --force cargo-make`
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:33]: `--force` is not needed with newer `cargo` versions

File: crate/guides/quickstart.md [hunk 2]
```diff
@@ -30,17 +30,18 @@ This framework requires you to install [Rust](https://www.rust-lang.org/tools/in
...
 
 ## The theoretical minimum
 
 To start, clone [the quickstart repo](https://github.com/seed-rs/seed-quickstart):
-`git clone https://github.com/seed-rs/seed-quickstart.git`,
-run `cargo make build` in a terminal to build the app, and `cargo make serve` to start a dev server
-on [127.0.0.1:8000](http://127.0.0.1:8000). If you'd like the compiler to automatically check and recompile when you 
-make changes, run `cargo make watch` instead of `cargo make build`.
+`git clone https://github.com/seed-rs/seed-quickstart.git`
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:38]: I would rather suggest to click on the green button `Use this template` in the quickstart repo GitHub page, so you can start to work on your project without quickstart's commit history. 
(Change it as you wish)

File: crate/guides/quickstart.md [hunk 3]
```diff
@@ -30,17 +30,18 @@ This framework requires you to install [Rust](https://www.rust-lang.org/tools/in
...
+
+To get a dev server deployed to [localhost:8000](http://127.0.0.1:8000), run `cargo make start`.
+
+`cargo make watch`is available for auto-magic re-builds
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:42]: `- available for auto-magic re-build` sounds scary :D 
What about something like - `- recompiles project when a Rust file is changed.` - or something like this.

File: crate/guides/quickstart.md [hunk 4]
```diff
@@ -30,17 +30,18 @@ This framework requires you to install [Rust](https://www.rust-lang.org/tools/in
...
+
+(Look through `Makefile.toml` for other project work-flow commands, e.g. release builds)
```
Comment (MartinKavik) [inline crate/guides/quickstart.md:44]: `Makefile.toml` - could you make it a link to that file?

File: crate/guides/quickstart.md (hunk 3 above)
Comment (Ben-PH) [reply crate/guides/quickstart.md:42]: it's more a personal flair. So I get a picture of what you're thinking, would this be appropriate?

` - recompiles the project auto-majically when a Rust file is changed.`?

**Pull request closed without merging**